library/billing.inc

```diff
@@ -229,40 +253,56 @@ function updateClaim(
         "x12_partner_id = '$partner_id'";
         ****/
         sqlBeginTrans();
-        $version = sqlQuery("SELECT IFNULL(MAX(version),0) + 1 AS increment FROM claims WHERE patient_id = ? AND encounter_id = ?", array( $patient_id, $encounter_id ));
+        $sqlBindArray = array();
+        $sqlBindArray[] = $patient_id;
+        $sqlBindArray[] = $encounter_id;
+        $version = sqlQuery("SELECT IFNULL(MAX(version),0) + 1 AS increment FROM claims WHERE patient_id = ? AND encounter_id = ?", $sqlBindArray);
+
+        $sqlBindArray = array();
+        $sqlBindArray[] = $patient_id;
+        $sqlBindArray[] = $encounter_id;
         if ($crossover<>1) {
             $sql .= "INSERT INTO claims SET " .
-              "patient_id = $patient_id, " .
-              "encounter_id = $encounter_id, " .
+              "patient_id = ?, " .
+              "encounter_id = ?, " .
               "bill_time = NOW() $claimset ," .
               "version = " . $version['increment'];
```

Comment:
``````
array_push($sqlBindArray, $patient_id, $encounter_id);
``````
And keep the line you have below.